src/irc/transport.rs

```diff
@@ -57,74 +67,80 @@ impl<S: AsyncRead + AsyncWrite> IrcServerConnection<S> {
                 v.extend_from_slice(line.as_bytes());
                 v.push(b'\n');
             }
-
-            if let Some(ref t) = inner.write_notify {
-                t.notify();
-            }
         }
-        self.poll_write().ok();
+
+        // ignore the result of the poll
+        self.write().await.ok();
```

Comment:
Instead of adding that comment, could you rewrite this and any other occurrences of the `<expr>.ok();` pattern to `let _ = <expr>;`, which is the standard way of saying "ignore result of <expr>" in Rust?